5.1.thats_the_way.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+FILE_DIDNT_EXIST_MSG = "The path does not exist or is not a folder"
+
+
+def is_exist_dir(folder):
+    """
+    :param folder: path to the folder
+    :return: True if the folder exists and is a folder
+    """
+    return os.path.exists(folder) and os.path.isdir(folder)
+
+
+def get_files_list(folder_path: str) -> list:
+    """
+    :param folder_path: path to the folder
+    :return: list of files in the folder that start with "deep"
+    """
+    # Get list of folders in the directory
```

Comment:
Remove these inline comments, they should be applied only when the code is unclear